### PR TITLE
Set up configuration and initialize connection to Kentix endpoints

### PR DESCRIPTION
**Pull request by zdevaty**

### CODE REVIEW

File: kentix/kentix.go [hunk 1]
```diff
@@ -59,31 +65,83 @@ func GetDeviceInfo(conf apiserver.Configuration) (*DeviceInfo, error) {
...
+	}
+	var sensorResponse sensorResponse
+	err = fetchData(url, &sensorResponse)
+	return &sensorResponse.Data, err
+}
+
 func fetchData(url string, dest interface{}) error {
 	client := &http.Client{}
 
 	req, err := http.NewRequest("GET", url, nil)
```
Comment (andreas-goebel) [inline kentix/kentix.go]: For an API access, please use our methods from go-utils/http. At the moment there is no method for authorization Basic with an API-Key, but I will create such a method. The difference between the API mock and real Kentic devices is, that the devices needs an authorization. You can see that in https://dev.azure.com/eliona/apps/_git/kentix?path=/kentix_doorlock_get_data.sh

The methods for go-utils/http can you find here: https://github.com/eliona-smart-building-assistant/go-utils/tree/main/http

Comment (andreas-goebel) [reply kentix/kentix.go]: There is already a method to do this:
http.NewRequestWithApiKey(url, "Authorization", "Basic " + API-Key from config in Base64)

Comment (andreas-goebel) [reply kentix/kentix.go]: Pleae note, that the  API-Key from config from config have to encoded in Base64. See: https://dev.azure.com/eliona/apps/_git/kentix?path=/kentix_doorlock_get_data.sh

Comment (andreas-goebel) [reply kentix/kentix.go]: With the method you can read the API in 2 lines:
request := http.NewRequestWithApiKey(url, "Authorization", "Basic " + conf.apiKey)
sensorData := http.Read[SensorData](request, 10, true)

**Review by andreas-goebel — CHANGES_REQUESTED** [review]
for API access, please use our methods from go-utils/http

File: kentix/kentix.go (hunk 1 above)
Comment (zdevaty) [reply kentix/kentix.go]: Thanks, I implemented it in the next CL.

**Review by andreas-goebel — APPROVED** [review]

**Pull request merged**